api/tests/src/logging/test_audit.py

```diff
@@ -141,7 +141,7 @@ def test_audit_hook(
         pass
 
     assert len(caplog.records) == len(expected_records)
-    for record, expected_record in zip(caplog.records, expected_records):
+    for record, expected_record in zip(caplog.records, expected_records, strict=True):
```

Comment:
Just a question, wondering why we are adding this? :)

Reply:
This is added because of a new lint rule